README.md

```diff
@@ -179,6 +180,12 @@ The difference between OpenAI-python v0 and v1 you may refer to the official doc
 
 ## Llama.cpp Inference
 
+### Llama.cpp Inference using Huggingface Tokenizer
+Example for inference using LLama-cpp-python can be found in: [llama_cpp_inference_example.py](llama_cpp_inference_example.py). 
+
+### Integration into Llama-cpp
+Besides, functionary was also integrated into LLama-cpp-python, however the integration might be not **quickly updated**, so if there is something wrong or weird in the result, please use: [llama_cpp_inference_example.py](llama_cpp_inference_example.py) instead.
+
```

Comment:
Maybe you can specifically explain that v2.5 is not integrated yet so just use the example script first?

Reply:
Ok, I added